ql-qlf-plugin/common/utils/Makefile_test.common

```diff
@@ -0,0 +1,92 @@
+# -*- Makefile -*-
+# Each plugin shall have a directory named 'test' that contains test cases
+# directories and a Makefile which includes this Makefile template.
+# The test Makefile specifies which tests to execute and how to verify them.
+# The test to execute should be explicitly specified in the TESTS variable.
+# Each test needs a verification step define in the name_of_test_verify variable.
+# A simple diff verification template have been defined in the template Makefile
+# diff_test performs a simple diff and takes name of file and its extension
+# Example of a test Makefile is given below:
+#
+# include $(shell pwd)/../../Makefile_test.common
+# TESTS = test1 test2
+# test1_verify = $(call diff_test,test1,ext) && test $$(grep "PASS" test1/test1.txt | wc -l) -eq 2
+# test2_verify = $(call diff_test,test2,ext)
+#
+GTEST_DIR = ../../../third_party/googletest/googletest
+CXX = $(shell yosys-config --cxx)
```

Comment:
Instead use conditional assignment for these variables.

Reply:
I modified the plugin folder structure to align on other plugins this time, we do not need modification to this anymore.